Makefile

```diff
@@ -225,3 +225,29 @@ serve-docs:
 	-it -p 4000:4000 \
 	jekyll/jekyll \
 	jekyll serve --livereload
+
+# gen-docs generates a new versioned docs directory under site/docs. It follows
+# the following process:
+#	1. Copies the contents of the most recently tagged docs directory into the new
+#	   directory, to establish a useful baseline to diff against.
+#	2. Adds all copied content from step 1 to git's staging area via 'git add'.
+#	3. Replaces the contents of the new docs directory with the contents of the
+#	   'master' docs directory, updating any version-specific links (e.g. to a
+#	   specific branch of the GitHub repository) to use the new version
+#   4. Copies the previous version's ToC file and runs 'git add' to establish
```

Comment:
These last 3 points are indented to the same level as the previous 3, is that intentional?

Reply:
not intentional, will fix